### PR TITLE
Add SortWithSwaps.

### PR DESCRIPTION
**Pull request by xuxiaoqiao**

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
+
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
```
Comment (xuxiaoqiao) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:10]: LGTM

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    private GameFramework framework = null;
+
+    // The last number clicked, which is to be swapped with the next number clicked
+    private int last = -1;
+
+    //The number of swaps the user has made
+    static Integer count = null;
+
+    public String getGameName() {
+        return new String("sort with swaps plugin");
```
Comment (xuxiaoqiao) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:29]: TEST

**Review by xuxiaoqiao — COMMENTED** [review]
TEST2

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 2 above)
Comment (LinSijie) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:29]: test 333

**Pull request merged**